rustls/src/common_state.rs

```diff
@@ -219,10 +221,15 @@ impl CommonState {
                 .pre_encrypt_action(f as u64)
             {
                 record_layer::PreEncryptAction::Nothing => {}
-                record_layer::PreEncryptAction::Close => {
-                    self.eager_send_close_notify(outgoing_tls)?;
-                    return Err(EncryptError::EncryptExhausted);
-                }
+                record_layer::PreEncryptAction::RefreshOrClose => match self.negotiated_version {
+                    Some(ProtocolVersion::TLSv1_3) => {
+                        self.refresh_traffic_keys_pending = Some(());
```

Comment:
The mutation on L327 is decorated with a comment motivating why this is setting a flag and expecting the caller to take the action. Worth duplicating that here?

Reply:
Added.